ci.cue

```diff
@@ -87,6 +89,43 @@ dagger.#Plan & {
 			command: flags: "-race": true
 		}
 
+		integration: bats.#Bats & {
```

Comment:
not important:

I think it would be nice to move this under test:

```cue
test: {
  unit: go.#Test ...
  
  integration: ...
```

So we can run `dagger do test integration`, or `dagger do test` to run all tests